Add a note about usecases for the clone and backup tools for 3.1 (#2191)
theforeman/foreman-documentation: guides/common/modules/con_backing-up-server-and-proxy.adoc

You can back up your {Project} deployment to ensure the continuity of your {ProjectName} deployment and associated data in the event of a disaster.
If your deployment uses custom configurations, you must consider how to handle these custom configurations when you plan your backup and disaster recovery policy.

> [NOTE]

Author: @evgeni does this note look good for the backup section?

You can back up your {Project} deployment to ensure the continuity of your {ProjectName} deployment and associated data in the event of a disaster.
If your deployment uses custom configurations, you must consider how to handle these custom configurations when you plan your backup and disaster recovery policy.
+
[NOTE]
====

> If you use the backup tool to create a new instance of the {ProjectServer}, the created instances are not supposed to run in parallel in a production environment.
> You must decommission the old instance after restoring the backup.

Reviewer: Can we unifiy this with the second note below?

```suggestion
The instances created using the backup tool are not supposed to run in parallel in a production environment.
You must decommission any old instances after restoring the backup.
```
